core/controllers/reader.py

```diff
@@ -721,16 +721,62 @@ class ReaderFeedbackHandler(base.BaseHandler):
 
     REQUIRE_PAYLOAD_CSRF_CHECK = False
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'subject': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': 'Feedback from a learner'
+            },
+            'feedback': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'has_length_at_most',
+                        'max_value': 10000
+                    }]
+                },
+                'default_value': None
```

Comment:
Why can this be `None`?

Reply:
I don't think there should be a default value.

Reply:
Thanks